app/code/Magento/Catalog/etc/di.xml

```diff
@@ -216,7 +216,7 @@
             <argument name="instanceName" xsi:type="string">Magento\Catalog\Model\System\Config\Source\Inputtype</argument>
         </arguments>
     </virtualType>
-    <virtualType name="Magento\Catalog\CategoryImageUpload" type="Magento\Catalog\Model\ImageUploader">
```

Comment:
I could only find this one usage of the virtual type though I thought there would be more.